lesson_3/les_3_task_1.py

```diff
@@ -0,0 +1,10 @@
+#В диапазоне натуральных чисел от 2 до 99 определить,
+# сколько из них кратны каждому из чисел в диапазоне от 2 до 9
```

Comment:
докстринги оформляются немного по другому
```suggestion
"""
В диапазоне натуральных чисел от 2 до 99 определить,
сколько из них кратны каждому из чисел в диапазоне от 2 до 9
"""
```